spec/helpers/session_helpers.rb

```diff
@@ -0,0 +1,25 @@
+module SessionHelpers
+
+  def sign_up(user)
+    visit '/'
+    fill_in 'email',    with: user.email
+    fill_in 'password', with: user.password
+    fill_in 'name',     with: user.name
+    fill_in 'username', with: user.username
+    click_button "Sign up"
+  end
+
+  def sign_in(user:, password:)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `user`. You can also write as `sign_in(*)` if you want the method to accept any arguments but don't care about them.<br>Lint/UnusedMethodArgument: Unused method argument - `password`. You can also write as `sign_in(*)` if you want the method to accept any arguments but don't care about them.